crates/cubecl-linalg/src/matmul/cmma/block_loop.rs

```diff
@@ -9,31 +9,142 @@ use super::{
     write_output::write_to_output,
 };
 
-#[cube]
+// #[cube]
+// pub(crate) fn block_loop<F: Float, FC: Float>(
+//     lhs: &Tensor<F>,
+//     rhs: &Tensor<F>,
+//     out: &mut Tensor<F>,
+//     mut offsets: Offsets,
+//     shared_memories: SharedMemories<FC>,
+//     accumulators: Accumulators<F>,
+//     #[comptime] config: CmmaConfig,
+//     dims: Dimensions,
+// ) {
+//     let block_size_k = config.block_size_k;
+//     let n_loops = (dims.k + block_size_k - 1) / block_size_k;
+
+//     for block in 0u32..n_loops {
+//         offsets.k = block * block_size_k;
+
+//         load_to_shared_memories::<F, FC>(lhs, rhs, offsets, shared_memories, dims, config);
+
+//         sync_units();
+
+//         compute_loop::<F, FC>(shared_memories, accumulators, config);
+
+//         sync_units();
```

Comment:
I guess it should be de-commented now, and the generated expand code bellow should be removed.